src/constants/common.ts

```diff
@@ -2,8 +2,7 @@ import { BigNumber, utils } from 'ethers';
 import { DistributionPeriodStage } from '../types/classes';
 
 // transaction management
-export const GAS_MULTIPLIER = 2;
-export const GAS_LIMIT_MAX = 1500000;
```

Comment:
Higher multiplier was needed for some `repayDebt` transactions.  `GAS_LIMIT_MAX` was not used anywhere.